_data/conferences.yml

```diff
@@ -23,6 +23,20 @@
 #
 ##########################
 
+- name: Mobile Conf Thailand
+  link: https://mobileconfth.com
+  start: 2019-08-24
+  end: 2019-08-24
+  location: 🇹🇭 Bangkok, Thailand
+  cocoa-only: false
+
+- name: Mobile Conf Thailand
+  link: https://mobileconfth.com
+  start: 2018-08-25
+  end: 2018-08-25
```

Comment:
Would it not be better to have one event from 24th to 25th?

Reply:
I've added two editions: one is 2018, the other is 2019, the days are close but it's a coincidence 😊